test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
   });
 });
 
 function exportNamespaces(...namespace) {
   let schemas = exportToSchemas(namespace);
+  expect(validateSchema(schemas));
```

Comment:
Also, `expect(validateSchema(schemas))` isn't really a complete thought.  The `validateSchema(schemas)` function returns a `ValidatorResult`.  So you need to be specific about what you expect.  For example:
```javascript
expect(validateSchema(schemas).valid).to.be.true;
// or
expect(validateSchema(schemas).errors).to.be.empty;
```